tools/authors.py

```diff
@@ -1,9 +1,11 @@
 #!/usr/bin/env python
 # -*- encoding:utf-8 -*-
 """
-git-authors [OPTIONS] REV1..REV2
+List the authors who contributed within a given revision interval::
 
-List the authors who contributed within a given revision interval.
+    python tools/authors.py REV1..REV2
+
+`REVx` being a commit hash.
```

Comment:
Better leave this out, since it's typically the tag rather than a commit hash. This is just the standard Git way of specifying a commit range.

Reply:
Looks like I mostly use hashes for the authors script and tags for the list script so at least for me I don't mind either way.